Kitodo/src/main/java/org/kitodo/production/services/command/MetadataScript.java

```diff
@@ -12,60 +12,62 @@
 package org.kitodo.production.services.command;
 
 import java.util.ArrayList;
+import java.util.HashMap;
 import java.util.List;
+import java.util.Map;
 import java.util.Objects;
 
+import org.apache.commons.lang.text.StrTokenizer;
+
 public class MetadataScript {
 
-    private String goal;
-    private String root;
+    private Map<String, String> parameters;
+    private String metadataKey;
+    private String valueSource;
     private String typeTarget;
+    private String variable;
     private List<String> values = new ArrayList<>();
```

Comment:
`values` is always zero or at most one element. Please change to `Optional<String> value;`